sound/soc/codecs/rt711-sdw.c

```diff
@@ -473,6 +480,8 @@ static int rt711_sdw_remove(struct sdw_slave *slave)
 
 static const struct sdw_device_id rt711_id[] = {
 	SDW_SLAVE_ENTRY_EXT(0x025d, 0x711, 0x2, 0, 0),
+	/* fake part to work-around missing _ADR in BIOS */
```

Comment:
This is ok, but just raising a note that it isn't immediately obvious "part" is referring to the "part" field of SDW. But yeah, people more familiar with will understand this for sure. Maybe "fake the part field to..." would be ever more clear.

Reply:
yes, I can improve it as 'fake part_id'